docs/subnets/deploy-a-gnosis-safe-on-your-evm.md

```diff
@@ -0,0 +1,810 @@
+# Deploy a Gnosis Safe on Your Subnet EVM
+
+## Introduction
+This article shows how to deploy and interact with a [Gnosis Safe](https://gnosis-safe.io/) programmatically on any [Subnet EVM](README.md).
+
+If you are looking for more information regarding the Gnosis Safe protocol, please check out [these developer docs](https://docs.gnosis-safe.io/).
+
+## Prerequisites
+This tutorial assumes that:
+
+- [A Subnet and EVM blockchain](./create-a-fuji-subnet.md) has been created.
```

Comment:
This links to Fuji subnet, however, later there is reference to local network. Please clarify which one or all of them can be used.

Reply:
Good cal